docs/trace/enriching-activity-processor/README.md

```diff
@@ -0,0 +1,113 @@
+# EnrichingActivityProcessor
+
+Sometimes the built-in instrumentation doesn't add enough data or you want to
+augment the data it provides with contextual information your application or
+library has available. For these cases OpenTelemetry-dotnet provides an
+`EnrichingActivityProcessor` and `EnrichmentScope` API.
+
+1. To enrich your spans first add the `EnrichingActivityProcessor` to your
+   `TracerProvider`:
+
+    ```csharp
+            using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+                .AddHttpClientInstrumentation()
+                .AddProcessor(new EnrichingActivityProcessor())
+                .AddConsoleExporter()
+                .Build();
+    ```
+
+    **Note:** Order is important. Make sure your `EnrichingActivityProcessor` is
+    registered before your `Exporter` and any other `ActivityProcessor`s you are
+    using.
+
+2. Use `EnrichmentScope.Begin` to wrap the call you want to instrument:
```

Comment:
For consideration: instead of using `2.`, use L2 heading (which helps to save the indentation and also gives anchor for people to link to.